MAINTAINERS

```diff
@@ -31,6 +31,78 @@ to appreciate the absence of bugs, the slow but steady improvement in stability,
 or the reliability of a release process. But those things distinguish a good
 project from a great one.
 """
+    [Rules.adding-maintainers]
+
+        title = "How are maintainers added?"
+
+        text = """
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to
+become maintainers are expected to be deeply involved in contributing code,
+pull request review, and triage of issues in the project for more than three
```

Comment:
So we don't need to review new issues? :-)
I'd just remove the "for more than three month" part of it.